aten/src/ATen/native/quantized/cpu/qnnpack/src/deconv-run.cc

```diff
@@ -0,0 +1,292 @@
+
+#include <cstring>
+#include <memory>
+
+#include <conv_utils.h>
+#include <pytorch_qnnpack.h>
+#include <qnnpack/indirection.h>
+#include <qnnpack/log.h>
+#include <qnnpack/math.h>
+#include <qnnpack/params.h>
+
+namespace qnnpack {
+struct q8conv_context {
+  size_t bs;
+  size_t ks;
+  size_t kc;
+  size_t kc_stride;
+  size_t m;
+  size_t m_stride;
+  size_t n;
+  size_t n_stride;
+  const uint8_t** indirect_a;
+  const void* packed_w;
+  uint8_t* c;
+  size_t c_stride;
+  union pytorch_qnnp_conv_quantization_params quantization_params;
+  const pytorch_q8conv_ukernel_function ukernel;
+};
+
+static void compute_q8conv(
```

Comment:
There is already a compute_q8conv in conv-run.cc. Lets use that if possible, please?

Reply:
factored out in #37497